### PR TITLE
Updated etcd version from v3.4.13 to v3.4.18

### PR DESCRIPTION
**Pull request by aaronfern**
**What this PR does / why we need it**:
This PR Upgrades the etcd version used from `v3.4.13` to `v3.4.18`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Upgraded etcd version from `v3.4.13` to `v3.4.18`
```

### CODE REVIEW

**Comment by ishan16696** [comment]
Can you also  update the release note.

**Comment by aaronfern** [comment]
> Can you also update the release note.
Done

**Review by abdasgupta — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v3.4.13-bootstrap-4
+v3.4.18
```
Comment (ishan16696) [inline VERSION:1]: can we also change the version with something appropriate as `v3.4.18` might be ambiguous as this is the custom image just like we used to add  `bootstrap` suffix

**Review by ishan16696 — CHANGES_REQUESTED** [review]

File: VERSION (hunk 1 above)
Comment (ishan16696) [reply VERSION:1]: Is this make sense ? @aaronfern @abdasgupta

Comment (aaronfern) [reply VERSION:1]: I think it's fine without the suffix as we're mirroring the etcd version in our custom image version
We could further add a suffix whenever we make changes to the bootstrap file

However, no strong opinion from me

**Review by ishan16696 — APPROVED** [review]

**Review by shreyas-s-rao — APPROVED** [review]

**Pull request merged**